netcontrold/tests/lib/test_prelim.py

```diff
@@ -0,0 +1,66 @@
+#
+#  Copyright (c) 2020 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from unittest import TestCase
+
+from netcontrold.lib import dataif
+from netcontrold.lib.error import ObjConsistencyExc
+
+class TestDataif_add_rxq(TestCase):
+
+	def setUp(self):
+		self.port1= dataif.Port("port1")
+
+
+	def test_port_add_rxq_1(self):
+		self.rxq1=self.port1.add_rxq(1)
```

Comment:
You would not need to store rxq in testcase instance as such, just to note down. It could just be a variable. Imagine the data structs that you would want to share between tests of this test suite, those could be stored with test instance.